data/scenarios/02Tutorials/04-build.yaml

```diff
@@ -0,0 +1,61 @@
+name: Build
+description: |
+  Learn how to build robots to carry out simple tasks.
+entities:
+  - name: goal
+    display:
+      attr: device
+      char: 'X'
+    description:
+    - Now that you know basic commands, it is time to
+      build robots to do the work for you.
+    - Let's start by building a one task gardener robot.
+    - You can build a robot with 'build {COMMANDS}',
+      where instead of 'COMMANDS' you write the sequence
+      of commands for robot to execute (separated by ';').
+    - Build a robot to grab the flower and place it next
+      to the water.
+
+win: |
+  def flowerAt = \p. teleport self p; ishere "flower"; end;
+  try {
+    teleport self (2,-1);
+    ishere "flower";
+    // BETTER:
+    // a <- flowerAt (2,-1);
+    // b <- flowerAt (1,-1);
+    // c <- flowerAt (1,-2);
+    // return (a || b || c);
```

Comment:
I have no idea why this does not work. When I manually tested it, it returned `true` just fine.